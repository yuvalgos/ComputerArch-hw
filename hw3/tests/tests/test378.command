./dflow_calc tests/test378.dat tests/test378.in p36 d193 p175 d35 p66 p3 p190 d101 d152 d166 p45 p77 p43 p140 p113 p52 d131 p117 p45 d7 d110 d56 d35 p168 p117 p37 p136 d135 p174 p191 p80 p47 d129 d176 p196 d29 p159 d78 p199 p29 d169 p117 p60 p73 d161 d16 p191 p195 d19 p146 d23 p108 d153 d116 d80 p164 d137 d162 p89 p94 d150 d157 p191 p83 p6 d63 d157 d75 p148 p23 p157 d114 p190 p139 p196 d109 d134 p187 p186 p142 p59 d112 d66 p44 p67 d163 p35 d81 d80 d120 d99 d126 p189 p9 p95 d43 p135 d101 p0 p4 d7 d93 p8