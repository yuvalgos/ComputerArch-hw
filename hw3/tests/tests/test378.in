8 24 28 23
4 18 13 2
7 29 1 1
6 30 27 4
1 3 8 3
4 23 11 31
0 31 8 21
7 24 15 0
3 17 26 28
2 19 15 30
3 11 4 2
2 9 31 8
8 25 29 0
8 30 17 7
5 23 15 11
2 17 29 22
8 11 28 28
3 8 8 10
7 26 6 13
2 5 20 9
0 4 16 19
5 19 11 11
2 14 17 21
2 20 31 10
4 1 26 0
3 31 20 10
7 29 14 7
3 19 2 31
8 2 9 15
7 15 7 1
5 18 11 26
3 19 15 12
8 19 22 26
5 19 26 28
5 23 27 27
0 3 11 28
8 7 13 22
3 10 7 7
4 11 6 30
2 16 30 25
2 10 5 18
2 27 23 21
6 23 2 14
4 15 7 12
2 30 27 24
3 19 30 31
3 7 15 5
2 15 29 12
0 13 1 3
0 23 11 6
3 31 9 14
3 11 22 5
5 1 5 1
4 7 27 23
6 22 23 10
2 8 26 23
2 25 7 11
3 30 9 5
2 4 20 4
2 0 11 4
5 22 9 6
3 20 14 15
8 30 20 17
6 21 27 21
1 27 15 20
5 13 17 15
8 13 13 18
0 9 19 19
6 31 22 20
6 4 4 12
2 14 6 7
7 0 0 22
6 14 30 22
3 17 24 17
5 29 28 29
2 21 19 6
3 21 24 25
4 23 27 2
2 15 15 10
0 3 9 25
4 26 24 23
8 4 21 2
8 18 26 21
4 15 15 11
5 21 18 0
2 13 16 22
5 19 13 28
0 26 5 24
2 14 3 10
8 23 28 23
5 1 18 9
3 7 10 31
2 1 25 21
6 4 15 2
6 18 0 10
8 29 28 13
8 27 11 20
8 2 0 3
0 22 12 27
0 17 5 7
3 0 16 16
6 2 3 11
2 12 0 25
7 9 17 30
4 11 23 16
5 20 9 13
4 29 7 19
6 1 23 29
4 28 22 16
5 30 31 0
4 4 31 21
5 26 26 15
6 18 13 12
3 4 2 0
7 24 15 18
8 18 1 30
5 0 19 20
3 8 20 31
4 14 4 18
0 28 21 26
5 10 20 9
0 11 2 7
2 24 18 25
3 6 16 6
8 12 6 19
7 14 24 9
1 14 31 7
6 8 6 17
5 6 6 30
2 31 10 2
2 29 3 24
6 27 0 11
4 14 20 8
2 15 2 16
1 7 27 15
1 2 28 12
4 25 11 27
1 14 7 4
7 15 20 6
4 19 29 8
1 23 3 16
1 31 1 2
8 14 10 23
3 29 1 13
2 22 4 27
6 18 12 8
5 2 10 8
8 14 7 25
5 12 27 27
8 8 23 19
2 27 15 25
8 24 30 17
4 5 17 0
2 11 8 11
0 19 20 9
1 24 25 11
8 23 31 26
7 24 30 15
7 26 10 2
7 23 13 5
1 14 16 18
1 1 29 27
4 26 2 19
2 31 17 25
2 0 12 12
4 16 17 28
5 13 31 25
8 31 29 20
0 19 20 1
7 14 30 0
6 10 0 17
8 6 5 11
0 17 0 22
6 16 8 19
7 10 17 28
2 14 15 9
1 2 29 26
5 29 11 27
0 7 20 1
0 0 4 22
7 6 8 29
8 5 11 15
1 12 16 17
7 13 20 27
1 18 0 29
2 2 31 5
0 5 19 23
2 16 30 22
0 9 26 29
1 5 28 23
1 0 4 23
2 21 15 2
6 16 30 14
2 14 4 8
5 7 22 13
3 29 27 28
6 28 3 15
0 31 24 27
5 1 11 6
2 1 25 5
