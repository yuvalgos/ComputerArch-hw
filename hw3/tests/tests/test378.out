Reading the opcodes latency info from tests/test378.dat ... Got latency for 9 opcodes
Reading the program file tests/test378.in ... Found 200 instructions
getProgDepth()==231
getDepDepth(36)==0
getInstDeps(193)=={113,149}
getDepDepth(175)==137
getInstDeps(35)=={16,-1}
getDepDepth(66)==66
getDepDepth(3)==0
getDepDepth(190)==161
getInstDeps(101)=={79,51}
getInstDeps(152)=={99,116}
getInstDeps(166)=={163,136}
getDepDepth(45)==24
getDepDepth(77)==63
getDepDepth(43)==23
getDepDepth(140)==61
getDepDepth(113)==78
getDepDepth(52)==30
getInstDeps(131)=={116,121}
getDepDepth(117)==118
getDepDepth(45)==24
getInstDeps(7)=={-1,-1}
getInstDeps(110)=={68,84}
getInstDeps(56)=={53,51}
getInstDeps(35)=={16,-1}
getDepDepth(168)==167
getDepDepth(117)==118
getDepDepth(37)==23
getDepDepth(136)==148
getInstDeps(135)=={119,124}
getDepDepth(174)==168
getDepDepth(191)==214
getDepDepth(80)==74
getDepDepth(47)==48
getInstDeps(129)=={120,101}
getInstDeps(176)=={143,162}
getDepDepth(196)==137
getInstDeps(29)=={-1,24}
getDepDepth(159)==142
getInstDeps(78)=={47,40}
getDepDepth(199)==164
getDepDepth(29)==30
getInstDeps(169)=={109,164}
getDepDepth(117)==118
getDepDepth(60)==7
getDepDepth(73)==24
getInstDeps(161)=={143,150}
getInstDeps(16)=={-1,-1}
getDepDepth(191)==214
getDepDepth(195)==164
getInstDeps(19)=={-1,11}
getDepDepth(146)==131
getInstDeps(23)=={6,-1}
getDepDepth(108)==85
getInstDeps(153)=={149,121}
getInstDeps(116)=={86,105}
getInstDeps(80)=={7,77}
getDepDepth(164)==161
getInstDeps(137)=={134,113}
getInstDeps(162)=={146,154}
getDepDepth(89)==74
getDepDepth(94)==53
getInstDeps(150)=={138,136}
getInstDeps(157)=={109,138}
getDepDepth(191)==214
getDepDepth(83)==58
getDepDepth(6)==0
getInstDeps(63)=={41,-1}
getInstDeps(157)=={109,138}
getInstDeps(75)=={45,-1}
getDepDepth(148)==148
getDepDepth(23)==2
getDepDepth(157)==137
getInstDeps(114)=={83,112}
getDepDepth(190)==161
getDepDepth(139)==147
getDepDepth(196)==137
getInstDeps(109)=={68,100}
getInstDeps(134)=={131,133}
getDepDepth(187)==153
getDepDepth(186)==169
getDepDepth(142)==131
getDepDepth(59)==29
getInstDeps(112)=={85,102}
getInstDeps(66)=={65,30}
getDepDepth(44)==19
getDepDepth(67)==31
getInstDeps(163)=={99,136}
getDepDepth(35)==23
getInstDeps(81)=={76,28}
getInstDeps(80)=={7,77}
getInstDeps(120)=={105,103}
getInstDeps(99)=={19,91}
getInstDeps(126)=={68,91}
getDepDepth(189)==161
getDepDepth(9)==17
getDepDepth(95)==52
getInstDeps(43)=={36,-1}
getDepDepth(135)==123
getInstDeps(101)=={79,51}
getDepDepth(0)==0
getDepDepth(4)==0
getInstDeps(7)=={-1,-1}
getInstDeps(93)=={83,28}
getDepDepth(8)==0
